extensions/3DTILES_metadata/schema/3DTILES_metadata.featureClass.property.schema.json

```diff
@@ -0,0 +1,72 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema",
+  "title": "Feature class property",
+  "type": "object",
+  "allOf": [{ "$ref": "glTFProperty.schema.json" }],
+  "properties": {
+    "name": {
+      "type": "string",
+      "minLength": 1,
+      "description": "The name of the property, e.g. for display purposes. The string is UTF-8 encoded."
+    },
+    "description": {
+      "type": "string",
+      "minLength": 1,
+      "description": "The description of the property. The string is UTF-8 encoded."
+    },
+    "elementType": {
+      "type": "string",
+      "enum": ["SCALAR", "ARRAY", "VARIABLE_SIZE_ARRAY", "STRING", "BINARY"],
+      "description": "The type of each element of the property array. This value may be a `SCALAR` (boolean, integer, or float types), or it may be an array type. If `SCALAR`, `ARRAY`, or `VARIABLE_SIZE_ARRAY` is used, then `componentType` must also be specified. `ARRAY` is a fixed-length array, and requires a `componentsPerElement` property. `VARIABLE_SIZE_ARRAY`, `STRING`, and `BINARY` also require a corresponding `offsetsBufferView` to indicate where each element starts in the buffer. `STRING` is similar to a `VARIABLE_SIZE_ARRAY` of `UINT8`, except that the contents will be a UTF-8 encoded string. `BINARY` is an alias for a `VARIABLE_SIZE_ARRAY` of `UINT8`. This adds semantic meaning that the element is used as a binary blob, rather than an array of numbers"
```

Comment:
Double check this line, the other day I made some wording changes because there are cases where fixed-length strings can be used.